x-pack/plugins/lens/public/datasources/form_based/operations/definitions/formula/formula.tsx

```diff
@@ -26,6 +26,7 @@ export interface FormulaIndexPatternColumn extends ReferenceBasedIndexPatternCol
   params: {
     formula?: string;
     isFormulaBroken?: boolean;
+    emptyAsNull?: boolean;
```

Comment:
I think this is a bit problematic because we don't have UI for this setting in Lens, so it will magically work and break as soon as the user goes to Lens.

Are you doing it to avoid showing a 0 in there if there are no errors?

If yes, why are you using a formula here, you should be able to do this with a normal quick function which has the setting exposed in the UI as well.